### PR TITLE
60-Min Blitz: What is PyTorch

### PR DESCRIPTION
**Pull request by CamiWilliams**
This update is in reference to the PyTorch Tutorial update strategy.

- Addressed issues #774 #775 #776 
- Made updates to the learning narrative of the tutorial

### CODE REVIEW

File: beginner_source/blitz/tensor_tutorial.py [hunk 1]
```diff
@@ -3,101 +3,147 @@
...
--  a deep learning research platform that provides maximum flexibility
-   and speed
+-  Those who are looking for a replacement for NumPy to use the power of GPUs.
+-  Researchers who want to build with a deep learning platform that provides maximum flexibility
+   and speed.
 
 Getting Started
 ---------------
 
+In this section of the tutorial, we will introduce the concept of a tensor in PyTorch, and it's operations.
```
Comment (jlin27) [inline beginner_source/blitz/tensor_tutorial.py]: Nit: it's --> its

File: beginner_source/blitz/tensor_tutorial.py [hunk 2]
```diff
@@ -3,101 +3,147 @@
...
+
 Tensors
 ^^^^^^^
 
-Tensors are similar to NumPy’s ndarrays, with the addition being that
-Tensors can also be used on a GPU to accelerate computing.
+A tensor is a generic n-dimensional array. tensors in PyTorch are similar to NumPy’s ndarrays,
```
Comment (jlin27) [inline beginner_source/blitz/tensor_tutorial.py]: Nit: Capitalize "tensors" in the second sentence

File: beginner_source/blitz/tensor_tutorial.py [hunk 3]
```diff
@@ -3,101 +3,147 @@
...
-###############################################################
-# .. note::
-#     An uninitialized matrix is declared,
-#     but does not contain definite known
-#     values before it is used. When an
-#     uninitialized matrix is created,
-#     whatever values were in the allocated
-#     memory at the time will appear as the initial values.
+"""
+We import ``future`` here to help port our code from Python 2 to Python 3.
```
Comment (jlin27) [inline beginner_source/blitz/tensor_tutorial.py]: Good call. Def was a common question from the workshop

File: beginner_source/blitz/tensor_tutorial.py [hunk 4]
```diff
@@ -3,101 +3,147 @@
...
+
 ###############################################################
-# Construct a tensor directly from data:
+# Let's construct a tensor with data that we define ourselves:
 
 x = torch.tensor([5.5, 3])
 print(x)
 
+"""
+Our tensor can represent all types of data. This data can be an audio waveform, the
```
Comment (jlin27) [inline beginner_source/blitz/tensor_tutorial.py]: +1

File: beginner_source/blitz/tensor_tutorial.py [hunk 5]
```diff
@@ -107,21 +153,29 @@
 # .. note::
 #     Any operation that mutates a tensor in-place is post-fixed with an ``_``.
 #     For example: ``x.copy_(y)``, ``x.t_()``, will change ``x``.
-#
-# You can use standard NumPy-like indexing with all bells and whistles!
+
+###############################################################
+# Similar to NumPy, tensors can be index using the standard
```
Comment (jlin27) [inline beginner_source/blitz/tensor_tutorial.py]: index --> indexed

File: beginner_source/blitz/tensor_tutorial.py [hunk 6]
```diff
@@ -107,21 +153,29 @@
...
+# Python ``x[i]`` syntax, where ``x`` is the array and ``i`` is the selection.
+# 
+# That said, you can NumPy-like indexing with all its bells and whistles!
```
Comment (jlin27) [inline beginner_source/blitz/tensor_tutorial.py]: you can <use> NumPy-like

File: beginner_source/blitz/tensor_tutorial.py [hunk 7]
```diff
@@ -107,21 +153,29 @@
...
 y = x.view(16)
 z = x.view(-1, 8)  # the size -1 is inferred from other dimensions
 print(x.size(), y.size(), z.size())
 
 ###############################################################
-# If you have a one element tensor, use ``.item()`` to get the value as a
-# Python number
+# You can access the Python number-value of a one-element tensor using ``.item()``.
+# If you have a multidimensional tensor, see the 
+# `tolist() https://pytorch.org/docs/stable/tensors.html#torch.Tensor.tolist` method.
```
Comment (jlin27) [inline beginner_source/blitz/tensor_tutorial.py]: I think you might need to add brackets and an underscore at the end for the hyperlink to work:
`tolist() <https://pytorch.org/docs/stable/tensors.html#torch.Tensor.tolist>`_

https://devguide.python.org/documenting/#hyperlinks
https://github.com/pytorch/tutorials/blob/master/advanced_source/dynamic_quantization_tutorial.py#L37

**Comment by jlin27** [comment]
Reads much more cohesively. Left some minor syntax notes. Thanks for the changes!

**Comment by CamiWilliams** [comment]
> Reads much more cohesively. Left some minor syntax notes. Thanks for the changes!

Addressed your comments!

**Pull request merged**